File: js_coding_q_and_a/05.js
// Remove duplicates from an array

var arr = [1, 2, 3, 2, 3, 1, 2, 3, 4, 4, 4, 3, 2, 4, 5, 5, 66, 98];

const removeDuplicates = (arr) => {
  var res = [];
  arr.forEach((num) => {
    if (!res.includes(num)) {
      res.push(num);
    }
  });
  return res;
};

// using set

arr = [...new Set(arr)];
console.log(arr);

// console.log(removeDuplicates(arr));

File: py_coding_q_and_a/05.py
# // Remove duplicates from an array

arr = [1, 2, 3, 2, 3, 1, 2, 3, 4, 4, 4, 3, 2, 4, 5, 5, 66, 98]

def remove_duplicates(arr):
    res=[]

    for num in arr:
        if num not in res:
            res.append(num)
    return res

print(remove_duplicates(arr))

# using set 
# x=set(arr)
# x=list(x)
# print(x)